code/modules/mob/holder.dm

```diff
@@ -32,6 +33,22 @@
 	for(var/mob/M in src.contents)
 		M.attackby(W,user)
 
+/obj/item/weapon/holder/attack_self(mob/M as mob)
+
+	if (contained && !(contained.stat & DEAD))
```

Comment:
Bitfield check still present.

`if (contained && contained.stat != DEAD)` is better.